packages/xdl/src/Env.ts

```diff
@@ -15,6 +15,13 @@ export function isLocal(): boolean {
   return getenv.boolish('EXPO_LOCAL', false);
 }
 
+export function getFeatureGateOverrides(): { enable: string[]; disable: string[] } {
+  return {
+    enable: getenv.array('EXPO_FG_ENABLE'),
```

Comment:
Traditionally our naming format for env variables in the CLI has been more like `EXPO_FG` and `EXPO_NO_FG` [ref](https://docs.expo.dev/workflow/expo-cli/#environment-variables)

Reply:
Yep, that makes sense for booleans, but this is to match the EAS cli `EAS_FG_ENABLE=test1,test2,test3` style which seemed best for string arrays.